main.js

```diff
@@ -0,0 +1,55 @@
+var board = [
+            ["1"],
+            ["2"],
+            ["5", "4", "3"]
+            ];
+
+var boardPrint = board.map(function(peg) {
```

Comment:
This should be a function so that whenever it is called, it can log the current state of the board. Here, it is just returning the initial state of the board.

You used map correctly, where most did not ;)

Reply:
Also it should console like this:

```
--- 5 4 3 2 1
---
---
```